scarlet/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "373409615880",
+    "project_id": "testowy-cf67a",
+    "storage_bucket": "testowy-cf67a.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:373409615880:android:5564b1bf3fecf87e8dc1bd",
+        "android_client_info": {
+          "package_name": "com.bijoysingh.quicknote.pro"
```

Comment:
This is config only for full/pro flavor. Lite flavor won't be able to use services.